prov/shm/src/smr_rma.c

```diff
@@ -142,7 +146,7 @@ ssize_t smr_generic_rma(struct smr_ep *ep, const struct iovec *iov,
 
 	cmd = ofi_cirque_tail(smr_cmd_queue(peer_smr));
 
-	if (cmds == 1) {
+	if (cmds == 1 && ep->region->cma_cap) {
```

Comment:
The cma_cap check should get added on line 130 where we calculate whether to go to the fast path or not

Reply:
Done.